json/it.json

```diff
@@ -247,60 +247,60 @@
     "If you did not request a password reset, no further action is required.": "Se non hai richiesto un reset della password, non è richiesta alcuna azione.",
     "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Se non hai un account, puoi crearne uno cliccando sul pulsante sotto. Dopo averlo creato, potrai cliccare il pulsante per accettare l'invito presente in questa email:",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se non puoi cliccare su \":actionText\", copia e incolla l'URL qui sotto\nnella barra degli indirizzi del tuo browser:",
-    "Increase": "Increase",
+    "Increase": "Aumenta",
     "India": "India",
     "Indonesia": "Indonesia",
     "Invalid signature.": "Firma non valida",
     "Iran, Islamic Republic Of": "Iran",
     "Iraq": "Iraq",
-    "Ireland": "Ireland",
-    "Isle Of Man": "Isle of Man",
-    "Israel": "Israel",
-    "Italy": "Italy",
-    "Jamaica": "Jamaica",
-    "January": "January",
-    "Japan": "Japan",
+    "Ireland": "Irlanda",
+    "Isle Of Man": "Isola di Man",
+    "Israel": "Israele",
+    "Italy": "Italia",
+    "Jamaica": "Giamaica",
+    "January": "Gennaio",
+    "Japan": "Giappone",
     "Jersey": "Jersey",
-    "Jordan": "Jordan",
-    "July": "July",
-    "June": "June",
-    "Kazakhstan": "Kazakhstan",
+    "Jordan": "Giordania",
+    "July": "Luglio",
+    "June": "Giugno",
+    "Kazakhstan": "Kazakistan",
     "Kenya": "Kenya",
```

Comment:
In exclude file ?

Reply:
Many countries have no translations. English "Kenya" is the same in Italian.

Reply:
If you like, I can add these exclusions to `script/excludes/it.php`.

Reply:
yes ;)

Reply:
Done ;)